pkg/registry/servicecatalog/instance/storage.go

```diff
@@ -144,7 +144,11 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage) {
 	statusStore := store
 	statusStore.UpdateStrategy = instanceStatusUpdateStrategy
 
-	return &store, &StatusREST{&statusStore}
+	referenceStore := store
+	referenceStore.UpdateStrategy = instanceReferenceUpdateStrategy
+
+	return &store, &StatusREST{&statusStore}, &referenceStore
```

Comment:
There needs to be a wrapper for referenceStore, that only presents `GET`, `PATCH`, and `UPDATE`, or the operations in the discovery endpoint will be wrong.  I think you can use `StatusREST` and it would work just fine.

Reply:
done